cache/redis_client.go

```diff
@@ -185,10 +193,28 @@ func NewRedisClient(logger log.Logger, name string, config RedisClientConfig, re
 	}
 	if config.MaxGetMultiConcurrency > 0 {
 		c.getMultiGate = gate.New(
-			prometheus.WrapRegistererWithPrefix("redis_getmulti_", reg),
+			prometheus.WrapRegistererWithPrefix(getMultiPrefix, reg),
```

Comment:
Same problem here. The final one will be `getmulti_cache_*` instead of `cache_getmulti_*`.

Reply:
Same reply on above https://github.com/grafana/dskit/pull/255#discussion_r1098156614